### PR TITLE
Fix tests for recursive spec file search

### PR DESCRIPTION
**Pull request by mfocko**
To be merged after packit/packit#1005

TODO:
- [x]  Add test for recursive spec file search

Signed-off-by: Matej Focko <mfocko@redhat.com>

### CODE REVIEW

File: tests/integration/test_issue_comment.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def test_issue_comment_propose_update_handler(
     )
     flexmock(
         GithubProject,
-        get_files=lambda ref, filter_regex: [],
+        get_files=lambda ref, filter_regex, recursive: [],
```
Comment (TomasTomecek) [inline tests/integration/test_issue_comment.py:112]: didn't you mean this? :)

```suggestion
        get_files=lambda ref, filter_regex: [], recursive,
```

Comment (mfocko) [reply tests/integration/test_issue_comment.py:112]: I didn't, or am I missing something? O.o
`get_files` takes ref, regex **and** whether it is recursive search

Comment (TomasTomecek) [reply tests/integration/test_issue_comment.py:112]: oh, I confused the syntax, yours is indeed correct

**Review by TomasTomecek — APPROVED** [review]
LGTM, thanks for a full test case

**Comment by mfocko** [comment]
recheck

**Pull request merged**